ocs_ci/utility/users.py

```diff
@@ -64,6 +64,41 @@ def delete_htpasswd_secret():
     exec_cmd(cmd)
 
 
+def delete_user(user_name=None):
+    """
+    Delete user
+
+    Args:
+        user_name (str): the user to delete
+    """
+    if user_name is None:
+        cmd = "oc delete user --all"
+    else:
+        cmd = f"oc delete user {user_name}"
+    try:
+        exec_cmd(cmd)
+    except Exception as e:
+        logging.error(e)
+
+
+def delete_identity(user_name=None):
+    """
+    Delete identity
+
+    Args:
+        user_name (str): the user to delete
+
+    """
+    if user_name is None:
+        cmd = "oc delete identity --all"
```

Comment:
I am not sure if deleting all identities is a good default behaviour when no parameter provided. Especially when it is not documented in docstring.

Reply:
done